AnnoyancesFilter/sections/popups.txt

```diff
@@ -89,6 +89,7 @@ sodapdf.com##.be-tooltip
 ! Specific sites
 !
 netfilmtvizle.com##body > #ics
+ifeng.com###root > a[class^="bookrack-"]
```

Comment:
I don't see it. Please make screenshot

Reply:
<details>
<summary>Screenshot </summary>
<img src="https://user-images.githubusercontent.com/18536328/76063129-28a82f80-5f98-11ea-8766-ecff7a6b0997.png" />
</details>